memory/hugepage_sanity.py.data/README.md

```diff
@@ -0,0 +1,20 @@
+Overview
+----------
+The hugepage_sanity test, tests the allocation of hugepages on Linux systems. It allocates a specified amount hugepages and then performs write operations to verify that the memory is accessible and functioning correctly. It primarily uses mmap system call and memset function to allocate the hugepages.
+
+
+Prerequisites
+--------------
+This test script requires a Linux system with hugepages support enabled. You must have the hugetlbfs supported in order to use hugepages.
+Additionally, you will need Python 3.x and the avocado-framework package installed on the Linux system.
```

Comment:
@Geet2 can we add how can we enable the hugepages support in shot, so that it will help new user to get this run without any issue

Reply:
Most linux systems have the hugepage support enabled by default.
There are _very few_ Linux systems  do not support HugePages by default. For such systems, the Linux kernel can be built using the CONFIG_HUGETLBFS and CONFIG_HUGETLB_PAGE configuration options. CONFIG_HUGETLBFS is located under File Systems and CONFIG_HUGETLB_PAGE is selected when you select CONFIG_HUGETLBFS.

Let me know, if you wnat me to add the above in the README file.

Reply:
mostly it is enabled by default, if not you need to build the kernel using the CONFIG_HUGETLBFS and CONFIG_HUGETLB_PAGE configuration options. CONFIG_HUGETLBFS is located under File Systems and CONFIG_HUGETLB_PAGE is selected when you select CONFIG_HUGETLBFS.